src/popup/components/AddressBook/AddressBookList.vue

```diff
@@ -63,19 +72,24 @@ export default defineComponent({
   },
   props: {
     modelValue: Boolean,
+    /** Whether the list is being used in an account selector or not  */
+    isSelector: Boolean,
```

Comment:
I think it's better to always emit event with the address and leave the decision what has to be done to parent component. In one case parent can `@something="router.push(...)"` and in other update the local state.